packages/core/src/application_ref.ts

```diff
@@ -732,6 +732,11 @@ export class ApplicationRef {
    */
   public readonly components: ComponentRef<any>[] = [];
 
+  /**
+   * Exposes an EnvironmentInjector that was used to create this application.
+   */
+  public readonly environmentInjector: EnvironmentInjector;
```

Comment:
Bike-shed: I wanted to propose the `injector` as a public name (and we already have a getter) to make it symmetric with other `*Ref`s. WDYT?
```suggestion
  public readonly injector: EnvironmentInjector;
```